src/jdk.incubator.code/share/classes/jdk/incubator/code/Op.java

```diff
@@ -591,8 +591,15 @@ public static Optional<FuncOp> ofElement(ProcessingEnvironment processingEnviron
             JavacScope scope = javacTrees.getScope(javacTrees.getPath(e));
             ClassSymbol enclosingClass = (ClassSymbol) scope.getEnclosingClass();
             FuncOp op = attr.runWithAttributedMethod(scope.getEnv(), methodTree,
-                    attribBlock -> reflectMethods.getMethodBody(enclosingClass, methodTree, attribBlock, make));
-            return Optional.of(op);
+                    attribBlock -> {
+                        try {
+                            return reflectMethods.getMethodBody(enclosingClass, methodTree, attribBlock, make);
+                        } catch (Throwable ex) {
```

Comment:
This avoids crashing if `reflectMethods` throws (which can happen if the source code we come from contains type checking errors -- such as calling methods with wrong arity)